chunky/src/java/se/llbit/chunky/world/WorldTexture.java

```diff
@@ -42,8 +44,11 @@
    * Set color at (x, z)
    *
    * @param frgb RGB color components
+   * @throws IllegalStateException if this WorldTexture has already been finalized.
    */
   public void set(int x, int z, float[] frgb) {
+    if (endFinalized) throw new IllegalStateException("Attempted to modify a finalized WorldTexture.");
+
```

Comment:
I don't think this is needed. Adding new tiles after "finalization" will just make it un-deduped. Deduplication isn't a strict requirement for using the world texture.

Reply:
Ok. I changed it so that if it is already "finalized", it will duplicate the texture (since it could be used by other chunks) and modify the new texture.